src/Razor/test/Microsoft.AspNetCore.Razor.Test.ComponentShim/Microsoft.AspNetCore.Components.netstandard2.0.cs

```diff
@@ -0,0 +1,841 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Components
```

Comment:
Spoke to @rynowak offline. Since this is not an exact copy-paste (some APIs were removed) from the ref assembly, we might have to do this again the next time we want to update the shim. This is fine for now but the next time we do this,  we should come up with an way of tracking what we want to delete when copy pasting content from the ref assembly.

Reply:
Yeah, the gist of it is that I removed the build-in components. We might make the decision that we want to leave them in too. I wasn't sure how that would affect tests so I took them out.